modules/homer/maketagdirectory/meta.yml

```diff
@@ -3,23 +3,34 @@ description: Create a tag directory with the HOMER suite
 keywords:
   - peaks
   - bed
+  - bam
+  - sam
 tools:
   - homer:
       description: |
         HOMER (Hypergeometric Optimization of Motif EnRichment) is a suite of tools for Motif Discovery and next-gen sequencing analysis.
       documentation: http://homer.ucsd.edu/homer/
       doi: 10.1016/j.molcel.2010.05.004.
       licence: ["GPL-3.0-or-later"]
+  - samtools:
+      description: |
+        SAMtools is a set of utilities for interacting with and post-processing
+        short DNA sequence read alignments in the SAM, BAM and CRAM formats, written by Heng Li.
+        These files are generated as output by short read aligners like BWA.
+      homepage: http://www.htslib.org/
+      documentation: hhttp://www.htslib.org/doc/samtools.html
+      doi: 10.1093/bioinformatics/btp352
+      licence: ["MIT"]
```

Comment:
I am not quite sure where to draw the line, but deseq2 has a doi and publication https://bioconductor.org/packages/release/bioc/html/DESeq2.html, so I would say it woudn't hurt to add it here

Reply:
Good catch!